README.md

```diff
@@ -7,185 +7,16 @@ The open-source RChain project is building a decentralized, economic, censorship
-[info] Compiling protobuf
-[info] Protoc target directory: /home/kirkwood/src/rchain/rspace/target/scala-2.12/src_managed/main
-[info] Compiling 18 Scala sources to /home/kirkwood/src/rchain/rspace/target/scala-2.12/classes ...
-[info] Done compiling.
-[success] Total time: 9 s, completed Feb 13, 2018 4:35:05 PM
-sbt:rspace> test
-[info] Packaging /home/kirkwood/src/rchain/rspace/target/scala-2.12/rspace_2.12-0.1.0-SNAPSHOT.jar ...
-[info] Done packaging.
-[info] Compiling 6 Scala sources to /home/kirkwood/src/rchain/rspace/target/scala-2.12/test-classes ...
-[info] Done compiling.
-[info] Packaging /home/kirkwood/src/rchain/rspace/target/scala-2.12/rspace_2.12-0.1.0-SNAPSHOT-tests.jar ...
-[info] Done packaging.
-[info] MultiplierUnitTests:
-[info] Multiplier tryParse
-[info] - should work
-[... eyerollingly thorough test output snipped...]
-[success] Total time: 7 s, completed Feb 13, 2018 4:35:13 PM
-sbt:rspace> 
-```
-
-#### Whole-project Build
-
-Everything that can be stitched together can be done so with the `node` project. `sbt` will even make a Docker image you can run. A single-line build command that does the trick is
-```
-<computer:~/src/rchain (dev)> sbt clean bnfc:clean bnfc:generate compile docker
-[... tons of output snipped ...]
-[info] Step 8/8 : ENTRYPOINT ["\/bin\/main.sh"]
-[info]  ---> Running in 2ac7f835192d
-[info] Removing intermediate container 2ac7f835192d
-[info]  ---> 5e79e6d92528
-[info] Successfully built 5e79e6d92528
-[info] Tagging image 5e79e6d92528 with name: coop.rchain/rnode
-[success] Total time: 35 s, completed May 24, 2018 10:19:14 AM
-```
-
-```
-<computer:~/src/rchain (dev)> docker images
-REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
-coop.rchain/rnode   latest              5e79e6d92528        7 minutes ago       143MB
-<none>              <none>              e9b49f497dd7        47 hours ago        143MB
-openjdk             8u151-jre-alpine    b1bd879ca9b3        4 months ago        82MB
-```
-
-### Running
+Assuming you have Docker running on your system, use the following pull command to fetch the latest version of RNode in Docker from the remote repository.
+`docker pull rchain/rnode:latest`
 
 Invoking the above Docker image is simple enough (exit with `C-c`):
 ```
-<computer:~/src/rchain (dev)> docker run -v $HOME/tmp:/var/lib/rnode -ti coop.rchain/rnode:latest
+<computer:~/src/rchain (dev)> docker run -v $HOME/tmp:/var/lib/rnode -ti -p 30304:30304 coop.rchain/rnode:latest --profile docker
```

Comment:
Line 188 - +<computer:~/src/rchain (dev)> docker run -v $HOME/tmp:/var/lib/rnode -ti -p 30304:30304 coop.rchain/rnode:latest --profile docker

Port should be 40400 per https://github.com/rchain/rchain/pull/1100